artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/ssl/SSLSupport.java

```diff
@@ -352,11 +352,14 @@ public static KeyStore loadKeystore(final String keystoreProvider,
       return ks;
    }
 
-   public static void checkPemProviderLoaded(String keystoreType) {
+   /**
+    * This method calls out to a separate class in order to avoid a hard dependency on the provider's implementation.
+    * This allows folks who don't use PEM to avoid using the corresponding dependency.
+    */
+   public static void checkPemProviderLoaded(String keystoreType) throws Exception {
```

Comment:
Is the signature reallyneeded anymore, now the reflection is gone?

Reply:
I'm not following. Why wouldn't it be needed?

Reply:
Since it wasnt needed beforehand and unlike the earlier reflection based changes adding various bits that throw, nothing seemingly now changes in the method in a way that freshly throws a checked exception?